docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -132,19 +181,25 @@ Step3: Configuring PacketFence
 PacketFence PKI
 ^^^^^^^^^^^^^^^
 
-To work with our PKI in PacketFence you will need to add the PKI in the PKI_Provider section.
+To work with our PKI in PacketFence you will need to add the PKI in the PKI Provider section.
 
 Click on add a new PKI and select PacketFence PKI.
 
 Follow the instructions below to fill the PacketFence PKI configuration.
 
-image::docs/images/packetfence-pki-provider.png [scaledwidth="100%",alt="PacketFence PKI configuration"]
+image::docs/images/packetfence-pki-provider.png[scaledwidth="100%",alt="PacketFence PKI configuration"]
+
+image::docs/images/packetfence-pki-provider2.png[scaledwidth="100%",alt="PacketFence PKI configuration"]
+
+Every fileds are explained in attached screenshot.
 
-+ explanation of screenshot
+Path need to be absolute (i.e. /usr/local/pf/conf/ssl/tls-cert/MyCA.pem is absolute, conf/ssl/tls-cert/MyCA.pem is relative)
 
-image::docs/images/packetfence-pki-provider2.png [scaledwidth="100%",alt="PacketFence PKI configuration"]
+You have to chose to use the onBoarding certificate using MacAddress or Username, this will define how you can revoke/deliver certificate. If you chose MacAddress, every user will need to register all of his devices using PacketFence onBoarding system, you can limit the number of max devices per user. If you revoke one certificate it will only block the device linked to this certificate. 
```

Comment:
How about we restart that whole section as Github will explode if I continue:
Proposition :

For the 'Common name Attribute', if you select 'Username', a certificate will be generated for the user that authenticated. If you select 'MAC address', a certificate will be generated for the device itself using the MAC address as the identifier.

This means that revoking the certificate for a username based certificate will block all the devices that this user registered. If you generate the certificates using the MAC address, revoking a certificate will block only this device.

If you want user based authentication then select 'Username'. If you prefer computer/device based authentication, select 'MAC address'